### PR TITLE
Simplify maya's Vagrantfile to help build & compile the maya repository

### PR DESCRIPTION
**Pull request by AmitKumarDas**
This PR fixes https://github.com/openebs/openebs/issues/568

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -40,53 +25,21 @@ sudo apt-get install -y zip unzip curl wget
...
-echo "export NOMAD_ADDR=http://172.28.128.3:4646" >> /home/vagrant/.profile
-
-SCRIPT
-
-
-$clientinstaller = <<SCRIPT
-
-sudo apt-get install -y open-iscsi fio
-
+echo -e "\n\tTIP: Start with command:- make"
```
Comment (prateekpandey14) [inline Vagrantfile]: `make` will only build maya-cli right now(GNUMakefile), have to add `make apisever` in Vagrantfile to builld m-apiserver.

Comment (prateekpandey14) [reply Vagrantfile]: by default `make` is running ` make test`

Comment (AmitKumarDas) [reply Vagrantfile]: @prateekpandey14 What should be the default target for make ?

**Comment by kmova** [comment]
can the title of the PR be changed..

**Comment by kmova** [comment]
Also, I need some instructions on how the developer will use this vagrant file. 

After compilation and CI is passed, what will be outcome? Can developer make some new changes to the code and re-compile?

**Comment by AmitKumarDas** [comment]
@kmova I will add a new issue to address your comments. It should be updated in the README.md file.
What do you say ?

File: Vagrantfile (hunk 1 above)
Comment (prateekpandey14) [reply Vagrantfile]: Sure, we can do this.

**Pull request merged**